IdentityCore/src/cache/MSIDMacTokenCache.m

```diff
@@ -276,6 +276,22 @@ - (BOOL)saveAppMetadata:(__unused MSIDAppMetadataCacheItem *)item
     return nil;
 }
 
+- (BOOL)saveOrUpdateMetadataItem:(NSData *)item
```

Comment:
does Mac token cache need to implement these?

Reply:
Not on this one, but on MSIDMacKeychainTokenCache,